loadgen/generic_executor.go

```diff
@@ -63,7 +64,8 @@ func (g *GenericExecutor) newRun(options RunOptions) (*genericRun, error) {
 
 // Run a scenario.
 // Spins up coroutines according to the scenario configuration.
-// Each coroutine runs the scenario Execute method in a loop until the scenario duration or max iterations is reached.
+// Each coroutine runs the scenario Execute method in a loop until the scenario duration or max
+// iterations is reached.
```

Comment:
I think line length of 120 is standard these days, can you make sure your editor supports that?

Reply:
It's not a matter of editor support, it's a matter of nicely fitting multiple buffers side by side. It's nice, I like it, and it's harmless. If it's something anyone cares about, stuff it in editorconfig or w/e. Shouldn't really be a thing anyone ever needs to comment on in a review.